### ISSUE TITLE
[1.21.3] In some cases, the blocks have no shadow.

### ISSUE THREAD

**Issue by Lolothepro** [neoforged/NeoForge#1613]
**Minecraft Version:** 1.21.2

**NeoForge Version:** 0-beta

**Logs:** https://mclo.gs/Oy3sCVR

**Steps to Reproduce:**
1. Load a world with mountains
2. See that there are no shadows on certain blocks (or take two screenshots, one Vanilla and one NeoForge, and compare)

**Description of issue:**

Vanilla:
![Image](https://github.com/user-attachments/assets/ea468f6f-32cb-434a-b3b3-aa468e0bc20a)

NeoForge:
![Image](https://github.com/user-attachments/assets/ea894090-34c8-4d9d-a0b4-3840cf40134b)

**Comment by Technici4n** [comment]
This is a consequence of us fixing a vanilla bug (#1370). Maybe we should revert this fix inside vanilla's AO code, since it is also fixed in the enhanced pipeline? Thoughts @embeddedt @XFactHD?

**Comment by embeddedt** [comment]
Yeah, we may as well revert it if the new pipeline is on by default.
